src/components/ProductCard.js

```diff
@@ -26,29 +29,63 @@ const wrapperStyles = {
   marginLeft: '2vw'
 };
 
-const Product_Card = (props)=>{
-  return(
-    <Col alignItems='center' style={{...wrapperStyles, backgroundColor: props.backgroundColor}}>
-      <Col alignItems='center'>
-        <img src={props.img} alt='' style={{ marginTop: 15, height: 95 }}/>
-      </Col>
+class ProductCard extends Component{
+  constructor(props){
+    super(props);
```

Comment:
When you are doing classes, let's use the short hand. Instead of 

```javascript
class Thing extends Component {
  constructor(props) {
    super(props)
    this.doThing = this.doThing.bind(this)
  }

  doThing() { console.log('thing') }
}
```

do this

```javascript
class Thing extends Component {
  doThing = () => { console.log('thing') }
}
```